memgpt/config.py

```diff
@@ -38,8 +33,7 @@ def set_field(config, section, field, value):
 
 @dataclass
 class MemGPTConfig:
-    config_path: str = os.path.join(MEMGPT_DIR, "config")
-    anon_clientid: str = None
+    anon_clientid: str = uuid.UUID(int=uuid.getnode()).hex
```

Comment:
as far as I can tell, a null anon_clientid is always wrong, so just setting a non-null default rather than having logic check it